src/sentry/static/sentry/app/views/settings/components/dataScrubbing/modals/form/sourceField.tsx

```diff
@@ -305,7 +306,7 @@ class SourceField extends React.Component<Props, State> {
     this.setState({showSuggestions});
   }
 
-  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+  handleChange = (_value: string, event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Please in this case, as we get the field's value in the first parameter, why did you prefer to use the event target value?

Reply:
changed. Was trying to keep lines changed down